bot/exts/events/hacktoberfest/_utils.py

```diff
@@ -0,0 +1,451 @@
+    in_review, accepted = await categorize_prs(bot, prs)
+
+    n = len(accepted) + len(in_review)  # Total number of PRs
+    if n >= PRS_FOR_SHIRT:
+        shirtstr = f"**{github_username} is eligible for hacktoberfest swag!**"
+    elif (remaining_prs := PRS_FOR_SHIRT - n) == 1:
+        shirtstr = f"**{github_username} is 1 PR away from being eligible for hacktoberfest swag!**"
+    else:
+        shirtstr = f"**{github_username} is {remaining_prs} PRs away from being eligible for hacktoberfest swag!**"
+
+    stats_embed = discord.Embed(
+        title=f"{github_username}'s Hacktoberfest",
+        color=Colours.purple,
+        description=f"{github_username} has made **{n}** valid {contributionator(n)} in October.\n\n{shirtstr}\n\n"
+    )
+
+    stats_embed.set_thumbnail(url=f"https://www.github.com/{github_username}.png")
+    stats_embed.set_author(
+        name="Hacktoberfest",
+        url="https://hacktoberfest.digitalocean.com",
+        icon_url="https://avatars1.githubusercontent.com/u/35706162?s=200&v=4"
+    )
+
+    # This will handle when no PRs in_review or accepted
+    review_str = build_prs_string(in_review, github_username) or "None"
+    accepted_str = build_prs_string(accepted, github_username) or "None"
+    stats_embed.add_field(
+        name=":clock1: In Review",
+        value=review_str
+    )
+    stats_embed.add_field(
+        name=":tada: Accepted",
+        value=accepted_str
+    )
+
+    logging.info(f"Hacktoberfest PR built for GitHub user '{github_username}'")
+    return stats_embed
+
+
+async def get_october_prs(bot: Bot, github_username: str) -> Optional[list[dict]]:
+    """
+    Query GitHub's API for PRs created by a GitHub user during the month of October.
+
+    PRs with an 'invalid' or 'spam' label are ignored unless merged or approved.
+
+    PRs have to be in a repository that has a 'hacktoberfest' topic,
+    unless the PR is labelled 'hacktoberfest-accepted' for it to count.
+
+    If PRs are found, return a list of dicts with basic PR information.
+
+    For each PR:
+    {
+        "repo_url": str
+        "repo_shortname": str (e.g. "python-discord/sir-lancebot")
+        "created_at": datetime.datetime
+        "number": int
+    }
+
+    Otherwise, return empty list.
+    None will be returned when the GitHub user was not found.
+    """
+    log.info(f"Fetching Hacktoberfest Stats for GitHub user: '{github_username}'")
+    base_url = "https://api.github.com/search/issues"
+
+    query_params = (
+        f"+type:pr"  # Only get PR if it's:
+        f"+is:public"  # - public,...
+        f"+author:{quote_plus(github_username)}"  # - by the user's github username,...
+        f"+-is:draft"   # - not a draft...
+        f"+created:{CURRENT_YEAR}-09-30T10:00Z..{CURRENT_YEAR}-11-01T12:00Z"  # and made in october.
+        f"&per_page=100"  # Limit results per-page returned from API (100 is the maximum)
```

Comment:
```suggestion
        f"+type:pr"  # Only get PR if it's:
        f"+is:public"  # public
        f"+author:{quote_plus(github_username)}"  # made by the user's github username
        f"+-is:draft"   # not a draft
        f"+created:{CURRENT_YEAR}-09-30T10:00Z..{CURRENT_YEAR}-11-01T12:00Z"  # made in October
        f"&per_page=100"  # Limit results per-page returned from API (100 is the maximum)
```
The `...` looks a bit redundant, up to you though.

Reply:
Agree, have removed. Also aligned the comments in a way that looks neater.